Assets/SteamVR_Unity_Toolkit/Scripts/Controls/2D/RadialMenuController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using UnityEngine;
+using System.Collections;
+using VRTK;
```

Comment:
I think all of these controls should probably be in the VRTK namespace as they are core scripts, then you wouldn't need to include VRTK.